src/app/components/AutoComplete.svelte

```diff
@@ -24,13 +36,17 @@
   // Index of focus element
   let index = -1;
 
+  onMount(async () => {
+    results = await worker.getDataFromStorage();
+  });
+
   // Input to search
-  function onChange(event: Event) {
+  async function onChange(event: any) {
```

Comment:
```suggestion
  function onChange(event: any) {
```

Also, why did the typing change from `Event` to `any`?

Reply:
I should have rebased production before requesting for a review, sorry for the confusion. 🤓